sktwocnt/lib/coordtrans.f90

```diff
@@ -146,54 +204,63 @@ subroutine coordtrans_ahlrichs2(c11, spheric, jacobi)
     !assert(size(c11) == 3)
     !assert(size(spheric) == 3)
 
-    rr = (zeta / log(2.0_dp)) * (1.0_dp + c11(1))**alpha &
-        &* log(2.0_dp / (1.0_dp - c11(1)))
+    rr = (zeta / log(2.0_dp)) * (1.0_dp + c11(1))**alpha * log(2.0_dp / (1.0_dp - c11(1)))
     spheric(1) = rr
     spheric(2) = acos(c11(2))
     spheric(3) = pi * (c11(3) + 1.0_dp)
 
-    jacobi = (zeta * (1.0_dp + c11(1))**alpha / log(2.0_dp)) &
-        &* (alpha * log(2.0_dp / (1.0_dp - c11(1))) / (1.0_dp + c11(1)) &
-        &+ 1.0_dp / (1.0_dp - c11(1))) * rr * rr * pi
+    jacobi = (zeta * (1.0_dp + c11(1))**alpha / log(2.0_dp)) * (alpha&
+        & * log(2.0_dp / (1.0_dp - c11(1))) / (1.0_dp + c11(1)) + 1.0_dp / (1.0_dp - c11(1)))&
+        & * rr * rr * pi
```

Comment:
Just a note: You don't have to use the 100-char line length in mathematical expressions, but can also break the line earlier, if it makes the expression easier to read. Whether the original or the newer is more readable, is of course a matter of taste :wink: